tests/resources/engine-scripts/check.pl

```diff
@@ -4,6 +4,10 @@
 use warnings;
 use Getopt::Long;
 
+if ($#ARGV < 1) {
+    die "The script must be used with one integer argument\n";
+}
```

Comment:
this is no more useful as we use real arguments with a parser for that.

Reply:
i ve removed it